internal/collector/capacity_scrape_test.go

```diff
@@ -1166,21 +1167,493 @@ func TestScanCapacityWithMailNotification(t *testing.T) {
+		))
+	}
+
+	// fill `project_az_resources` with some usage data
+	// (we want to see how commitment confirmation reacts to existing usage)
+	berlin := s.GetProjectID("berlin")
+	dresden := s.GetProjectID("dresden")
+	paris := s.GetProjectID("paris")
+	firstCapacityAZOne := s.GetAZResourceID("first", "capacity", "az-one")
+	firstCapacityAZTwo := s.GetAZResourceID("first", "capacity", "az-two")
+
+	committedForTenDays := must.Return(limesresources.ParseCommitmentDuration("10 days"))
+	committedForTwentyDays := must.Return(limesresources.ParseCommitmentDuration("20 days"))
+	const oneDay = 24 * time.Hour
+
+	// now we place a commitment in one project to be transferred and another one, but in different AZs
+	expiry := s.Clock.Now().Add(10 * oneDay)
+	creation := s.Clock.Now()
+	UUID1 := add(db.ProjectCommitment{
+		UUID:              s.Collector.GenerateProjectCommitmentUUID(),
+		ProjectID:         berlin,
+		AZResourceID:      firstCapacityAZOne,
+		Amount:            1,
+		CreatedAt:         creation,
+		Duration:          committedForTenDays,
+		TransferToken:     Some(s.Collector.GenerateTransferToken()),
+		TransferStatus:    limesresources.CommitmentTransferStatusPublic,
+		TransferStartedAt: Some(s.Clock.Now()),
+	})
+	UUID2 := add(db.ProjectCommitment{
+		UUID:         s.Collector.GenerateProjectCommitmentUUID(),
+		ProjectID:    dresden,
+		AZResourceID: firstCapacityAZTwo,
+		Amount:       1,
+		CreatedAt:    s.Clock.Now(),
+		Duration:     committedForTenDays,
+	})
+	tr.DBChanges().Ignore()
+
+	s.Clock.StepBy(1 * time.Hour)
+	mustT(t, jobloop.ProcessMany(job, s.Ctx, len(s.Cluster.LiquidConnections)))
+
+	// both commitments get confirmed
+	now := s.Clock.Now().Add(-5 * time.Second)
+	confirmation := now
+	tr.DBChanges().AssertEqualf(`
+		UPDATE project_az_resources SET quota = 1 WHERE id = 13 AND project_id = 2 AND az_resource_id = 3;
+		UPDATE project_az_resources SET quota = 1 WHERE id = 2 AND project_id = 1 AND az_resource_id = 2;
+		UPDATE project_commitments SET status = 'confirmed', confirmed_at = %[1]d WHERE id = 1 AND uuid = '%[2]s' AND transfer_token = 'dummyToken-1';
+		UPDATE project_commitments SET status = 'confirmed', confirmed_at = %[1]d WHERE id = 2 AND uuid = '%[3]s' AND transfer_token = NULL;
+		UPDATE project_resources SET quota = 1 WHERE id = 1 AND project_id = 1 AND resource_id = 1;
+		UPDATE project_resources SET quota = 1 WHERE id = 5 AND project_id = 2 AND resource_id = 1;
+		UPDATE project_services SET quota_desynced_at = %[1]d WHERE id = 1 AND project_id = 1 AND service_id = 1;
+		UPDATE project_services SET quota_desynced_at = %[1]d WHERE id = 3 AND project_id = 2 AND service_id = 1;
+		%[4]s
+	`, now.Unix(), UUID1, UUID2, timestampUpdates())
+
+	// now we place a commitment that is in the same project, so it cannot be consumed by the transfer one
+	// this situation can only occur, if the customer wants to get rid of a commitment, which is older than one they placed later
```

Comment:
The first sentence is phrased the wrong way around. In the second sentence, "can only occur" is misleading. The sentence describes the specific problematic situation where this arrangement could arise.
```suggestion
	// now we place a commitment that is in the same project, so it cannot be consume the transferable one;
	// this checks that we avoid the loophole where the customer wants to get rid of an
	// old undeleteable commitment by having it be consumed by a newer deleteable one
```